jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/BuildImageMojoTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.maven;
+
+import java.io.IOException;
+import java.util.logging.Logger;
+import org.apache.maven.it.VerificationException;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+/** Tests for {@link BuildImageMojo}. */
+public class BuildImageMojoTest {
+
+  @ClassRule public static final TestPlugin testPlugin = new TestPlugin();
+
+  @Rule public final TestProject testProject = new TestProject(testPlugin);
+
+  private static Logger log = Logger.getLogger("info");
+
+  @Test
+  public void testExecute() throws VerificationException, IOException {
+    //    Verifier verifier = new Verifier(testProject.getProjectRoot().toString());
```

Comment:
Still need to fix up the plugin test. Will probably need to move this to be an integration test and test against a local registry.